Source/bmalloc/libpas/src/libpas/pas_enumerator.c

```diff
@@ -462,4 +464,92 @@ bool pas_enumerator_enumerate_all(pas_enumerator* enumerator)
     return true;
 }
 
+static crash_reporter_memory_reader_t memory_reader;
+
+static kern_return_t
+memory_reader_adapter(task_t task, vm_address_t address, vm_size_t size, void **local_memory)
+{
+    void *ptr = memory_reader(task, address, size);
+    *local_memory = ptr;
```

Comment:
Can a `nullptr` be passed for `local_memory`?  If so, this needs a `nullptr` check:

```
    if (local_memory)
        *local_memory = ptr;
```

Reply:
I'll add the check.

You could theoretically pass a nullptr.